package.json

```diff
@@ -14,8 +14,10 @@
   },
   "scripts": {
     "start": "node server/node/server.js",
+    "dev": "node server/node/server.js & npm run webhook",
     "setup-products": "node server/node/setup.js",
-    "postinstall": "npm run setup-products"
+    "webhook": "stripe listen --project-name=stripe-payments-demo --forward-to http://localhost:8000/webhook",
+    "postinstall": "brew install stripe/stripe-cli/stripe && npm run setup-products"
```

Comment:
What happens on platforms where `brew` isn't available? Do we expect this project to only be run on macOS laptops with Homebrew already installed?

Reply:
@ob-stripe good point. Do you think it's better to link to https://github.com/stripe/stripe-cli#installation from the README instead? I still think it's peaceful to include because it would simply error the first script, but still run the second script, no?

Reply:
I've updated the READMEs with installation instructions.